.github/ISSUE_TEMPLATE/sample_app_proposal.yaml

```
@@ -0,0 +1,50 @@
+name: Sample App Proposal
+description: Show off your app!
```

```suggestion
name: Sample App Proposal
description: Show off your app!
title: "[Sample App Proposal] <title>"
labels: Sample Proposal
```

It would be great to add title and labels to manage our issues :smile: 

you can check title of bug ISSUE TEMPLATE:
https://raw.githubusercontent.com/euphony-io/euphony/master/.github/ISSUE_TEMPLATE/bug-report.yaml

you can check euphony's labels at link:
https://github.com/euphony-io/euphony/labels

---

Thanks for feedback!! I applied your feedback :)